api/examples/pod.json

```diff
@@ -11,7 +11,7 @@
         "image": "dockerfile/nginx",
         "ports": [{
           "containerPort": 80,
-          "hostPort": 8080
+          "hostPort": 8081
```

Comment:
Is it possible to do this without using a host port? Services have generally replaced the need for host ports, and using host ports mean that the scheduler can't put two instances of the same pod onto a single node.